.gitignore

```diff
@@ -105,6 +105,8 @@ tests/testdata/raster/band3_float32_noct_epsg4326.tif.aux.xml
 tests/testdata/raster/band3_int16_noct_epsg4326.tif.aux.xml
 tests/testdata/tenbytenraster.asc.aux.xml
 tests/testdata/test_plugin_path/plugin_started.txt
+tests/testdata/test_qgis_config_path/profiles/default/*
+!tests/testdata/test_qgis_config_path/profiles/default/python
```

Comment:
Just checking, is this change accidental?

Reply:
A new test introduced the load of user expressions (from QGIS_CUSTOM_CONFIG_PATH).

This `.gitignore` change aims to ignore everything from that directory (e.g., fonts, .db and .ini files) but the `profiles/default/python/` directory content, which corresponds to the user expressions.

Reply:
OK